coding_test.go

```diff
@@ -0,0 +1,53 @@
+package merkledag_test
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	cid "github.com/ipfs/go-cid"
+	ipld "github.com/ipfs/go-ipld-format"
+	"github.com/ipfs/go-merkledag"
+)
+
+var benchInput []byte
+
+func init() {
+	someData := bytes.Repeat([]byte("some plaintext data\n"), 10)
+	someCid, _ := cid.Cast([]byte{1, 85, 0, 5, 0, 1, 2, 3, 4})
```

Comment:
what is this and why are we skipping the error check?

Reply:
as I understand it, all we're trying to do here is make a test protonode with 10 links, and we don't really care about the content. @mvdan wrote this code though

Reply:
yep it's just a sample dummy CID for use in tests.

Reply:
seems fine, but some explanation would be great (i.e. I think this is a CIDv1 with a raw codec, and the identity multihash with length 5 and the value `[]{0, 1, 2, 3, 4}`).